doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -2025,7 +2027,24 @@ curl -X GET -u <user>:<password> https://localhost:4000/api/api-key -k
 
 ```
 
-#### 19.1.3 Revoking API keys
+#### 19.1.3 Updating API keys
+
+An existing API key is updated with a `PUT` request to `/api/api-key/{id}`. Message body must contain the roles to be
+associated with the key. Server responds with data that contains the key id and roles associated with the key.
+
+```
+curl -X PUT -u <user>:<password> https://localhost:4000/api/api-key/60 --data '["XROAD_SECURITYSERVER_OBSERVER","XROAD_REGISTRATION_OFFICER"]' --header "Content-Type: application/json" -k
+{
+  "id": 61,
```

Comment:
Looks like update targets key 60 but returns 61, is it a mistake?

Reply:
This was a mistake. It's fixed now.